### PR TITLE
Fix specific import examples in the README

### PR DESCRIPTION
**Pull request by davidfou**
Hi,

I think some import are broken in the README...

David

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ But this should throw the `only one of the following may be provided` error
 
 The possible solution
 ```js
-import { singlePropFrom } from 'react-prop-types/singlePropFrom';
+import { singlePropFrom } from 'react-prop-types/lib/singlePropFrom';
```
Comment (taion) [inline README.md]: This should just be `import singlePropFrom`

**Comment by taion** [comment]
Nice catch, thanks. Can you address the one nit I left?

**Comment by davidfou** [comment]
Done!

**Comment by jquense** [comment]
thanks

**Pull request merged**